config/config.yaml

```diff
@@ -241,6 +250,12 @@ cluster:
     # We encourage you NOT to change this parameter unless you know what you are doing.
     tick_period_ms: 100
 
+    # Number of replicas of each shard that network tries to maintain
+    replication_factor: 1
+
+    # How many replicas should apply the operation for us to consider it successful
+    write_consistency_factor: 1
```

Comment:
Not sure if those should be part of the `ConsensusConfig`. It is not a part of `StorageConfig`, so it would require to change a lot of code to pass it deep enough to use in https://github.com/qdrant/qdrant/blob/28dee3821f473a4cc15875ef8e4652e52e4cf9ed/lib/storage/src/content_manager/toc/create_collection.rs#L25

Reply:
How about putting them under `storage.collection` or `storage.collection_params`?